src/main/resources/db/migration/common/V1_0_50__add-kpi-column-fixed-user-id.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE kpi
+    ADD COLUMN IF NOT EXISTS fixed_user_id VARCHAR(64);
```

Comment:
Could this column already exist?

Reply:
Why is this column even necessary for the removal of the key identifier?

Reply:
The source of this db change is VACCINECER-1956 Anwendung Schema-Anpassungen. Harald has taken it into his branch to prevent file naming conflicts with his own DB-changes when merged. This is the only change for VACC-1956.